src/Microsoft.AspNet.Mvc.TagHelpers/Internal/JavaScriptStringArrayEncoder.cs

```diff
@@ -1,43 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System.IO;
-using System.Text.Encodings.Web;
-
-namespace Microsoft.AspNet.Mvc.TagHelpers.Internal
-{
-    /// <summary>
-    /// Methods for encoding <c>string[]</c>s for use as a JavaScript array literal.
-    /// </summary>
-    public static class JavaScriptStringArrayEncoder
-    {
-        /// <summary>
-        /// Encodes a <c>string[]</c> for safe use as a JavaScript array literal in many contexts, including
-        /// inline in an HTML file.
-        /// </summary>
-        public static string Encode(JavaScriptEncoder encoder, string[] values)
```

Comment:
Don't think I noticed this change before. Why not pass the `TagHelperContent` into this method? Nice to keep the `BuildFallbackBlock()` length down somewhat.